saleor/graphql/schema.graphql

```diff
@@ -3549,7 +3562,7 @@ type ProductVariant implements Node & ObjectWithMetadata {
   images: [ProductImage]
   translation(languageCode: LanguageCodeEnum!): ProductVariantTranslation
   digitalContent: DigitalContent
-  stock(country: String): [Stock]
+  stocks(country: String): [Stock]
```

Comment:
Why not `CountryCode`?

Reply:
:information_desk_person: I can change if you want.

Reply:
Good catch, we have an enum for that.